server/app/views/components/FieldWithLabel.java

```diff
@@ -443,7 +455,24 @@ public DivTag getTextareaTag() {
       if (this.cols.isPresent()) {
         textareaFieldTag.withCols(String.valueOf(this.cols.getAsLong()));
       }
-      return applyAttrsAndGenLabel(textareaFieldTag);
+      return applyAttrsClassesAndLabel(textareaFieldTag);
+    }
+
+    throw new RuntimeException("needs to be textarea tag");
+  }
+
+  public DivTag getUSWDSTextareaTag() {
```

Comment:
This is also doing almost the same things as getTextareaTag(). Can you look to combining them?

Reply:
I see what you're saying.  I considered the option of passing in a `isUSWDS` variable here, but there are 21 usages of this method throughout the app and I'd have to add the parameter in all of those places.  Then, when USWDS becomes the norm for our input fields, we'd have to remove the parameters in all those places.  By duplicating the function, we temporarily have two almost identical functions, but we'll have less to change when we are only using USWDS for input fields.  We'll be able to blend the two functions into one at that time.  Does that sound reasonable?

Reply:
ok sounds good! - Can you add a TODO item to make sure we followup. Thank you.

Reply:
Good idea, will do!

Reply:
Done